package com.gilbersoncampos.testeaiko.exception

sealed class CustomException (message:String):Exception(message){
    data object NotFoundException:CustomException("NÃ£o encontrado")
    data object ErrorException:CustomException("Error")
}